src/posit/connect/content.py

```diff
@@ -27,6 +41,12 @@ def __getitem__(self, key: Any) -> Any:
             return ContentItemOwner(params=self.params, **v)
         return v
 
+    @property
+    def oauth(self) -> ContentItemOAuth:
+        if "guid" not in self:
+            raise ValueError("ContentItemOAuth requires content guid")
```

Comment:
I think you can remove this. If 'guid' is missing, we have bigger issues. And I believe the default KeyError thrown by self['guid'] if 'guid' is missing is sufficient.

Reply:
cool, yep - makes sense. I was getting type checking errors when we were using property-based fields because the guids were `str|None` types, but the change to a `self['guid']` lookup removes that issue entirely. 

I'll make that change and then merge when CI passes.